### PR TITLE
avant dernière pr dev dans main

### PR DESCRIPTION
**Pull request by pjroggy**

### CODE REVIEW

**Review by PaulBOURY — APPROVED** [review]

**Review by JadeJolivet — APPROVED** [review]
square like a watermelon

**Review by FGAT3 — APPROVED** [review]
Square like pasteque

File: client/src/main.jsx [hunk 1]
```diff
@@ -88,21 +137,19 @@ const root = ReactDOM.createRoot(document.getElementById("root"));
...
+      autoClose={3000}
+      hideProgressBar={false}
+      newestOnTop={false}
+      closeOnClick
+      rtl={false}
+      pauseOnFocusLoss
+      draggable
+      pauseOnHover
+      theme="light"
+      transition:Bounce
```
Comment (Yoann-Fortin) [inline client/src/main.jsx]: Bounce is not imported, are you sure it works?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/main.jsx (hunk 1 above)
Comment (pjroggy) [reply client/src/main.jsx]: No import it's the setting of the ToastContainer, all import with toastify

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**